### PR TITLE
Bjorncs/accepted ciphers

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: security-utils/src/test/java/com/yahoo/security/tls/TransportSecurityOptionsTest.java [hunk 1]
```diff
@@ -16,19 +18,20 @@
...
+        TransportSecurityOptions expectedOptions = new TransportSecurityOptions("myhost.key", "certs.pem", "my_cas.pem", CIPHERS);
         TransportSecurityOptions actualOptions =  TransportSecurityOptions.fromJsonFile(TEST_CONFIG_FILE);
         assertEquals(expectedOptions, actualOptions);
     }
 
     @Test
     public void can_read_options_from_json() throws IOException {
         String tlsJson = new String(Files.readAllBytes(TEST_CONFIG_FILE), StandardCharsets.UTF_8);
-        TransportSecurityOptions expectedOptions = new TransportSecurityOptions("myhost.key", "certs.pem", "my_cas.pem");
+        TransportSecurityOptions expectedOptions = new TransportSecurityOptions("myhost.key", "certs.pem", "my_cas.pem", CIPHERS);
```
Comment (vekterli) [inline security-utils/src/test/java/com/yahoo/security/tls/TransportSecurityOptionsTest.java]: Update `equals`, `hashCode` and `toString` in `TransportSecurityOptions` as well?

**Review by vekterli — APPROVED** [review]
:+1: but see comment re: equals etc

**Comment by vekterli** [comment]
For the record, we'll be using IANA cipher names in the config file. We'll remap these to OpenSSL when needed (explicit cipher set is not supported there yet).

**Review by vekterli — APPROVED** [review]

**Comment by hmusum** [comment]
There is a merge stop

**Pull request merged**